packages/cardpay-reward-programs/cardpay_reward_programs/rule.py

```diff
@@ -30,8 +29,8 @@ def set_user_defined_parameters(
     def sql(self, table_query):
         raise NotImplementedError
 
-    def _get_table_query(self, table_name, min_partition: int, max_partition: int):
-        config_location = self.subgraph_config_location[table_name]
+    def _get_table_query(self, config_name, table_name, min_partition: int, max_partition: int):
```

Comment:
The table name and which subgraph config part to extract are not the same.

Reply:
What do you mean by "subgraph_config_part"?

Reply:
Yeah that sentence didn't make much sense

So if a rule did something using the rewards extracted data and the payments, it might have:

```
"subgraph_config_locations": {
    "prepaid_card_payment": "s3://...",
    "rewards": "s3://..."
}
```

Then the "config_name" (we could come up with a better name) here would be "prepaid_card_payment" or "rewards". Prepaid card payment example we have has just one table in it, but others (like the rewards one) have multiple.

Reply:
Oh ok. Got it.